modules/load_medical_images.ipynb

```diff
@@ -284,9 +347,16 @@
     "And use the meta data of the first image to represent the stacked result."
```

Comment:
Please also change the title of this cell from `1 training item` to `a multi-channel image`.

Thanks.